protobuf-definitions/proto/mlagents_envs/communicator_objects/observation.proto

```diff
@@ -19,4 +19,5 @@ message ObservationProto {
         bytes compressed_data = 3;
         FloatData float_data = 4;
     }
+    repeated int32 mapping = 5;
```

Comment:
Please rename to something more descriptive, such as `compressed_channel_mapping`.

I think you also need to document the format of this in more detail, either here or in the GetCompressionMapping() docstring.

Reply:
Plan to put it in GetCompressionMapping()